lib/Serialization/ModuleFile.cpp

```diff
@@ -520,12 +495,10 @@ void ModuleFile::getImportDecls(SmallVectorImpl<Decl *> &Results) {
                  "deserialized imports should not be ambiguous");
           Kind = *FoundKind;
         }
-
-        AccessPath.push_back({ ScopeID, SourceLoc() });
       }
 
       auto *ID = ImportDecl::create(Ctx, FileContext, SourceLoc(), Kind,
-                                    SourceLoc(), AccessPath);
+                                    SourceLoc(), importPath.get());
```

Comment:
The section between line 481/464 and line 528/501 has been significantly restructured and deserves special attention.

Reply:
@xymus I'd appreciate if you could review at least this portion of the PR (plus the other one I'll ping you in).

Reply:
Same here. 👍